adapters/tlsio_openssl.c

```diff
@@ -88,6 +88,7 @@ struct CRYPTO_dynlock_value
 
 static const char* const OPTION_UNDERLYING_IO_OPTIONS = "underlying_io_options";
 #define SSL_DO_HANDSHAKE_SUCCESS 1
+static int g_ssl_cert_max_size_in_mb = 10;
```

Comment:
>g_ssl_cert_max_size_in_mb [](http://example.com/codeflow?start=11&length=25)

Is it certificate size or certificate revocation list (CRL) size ?

Reply:
it is certificate size

Reply:
It's the CRL Size.

---
In reply to: [616126867](https://github.com/fmegen/azure-c-shared-utility/pull/54#discussion_r616126867) [](ancestors = 616126867)